Misc/stable_abi.toml

```diff
@@ -2528,6 +2528,26 @@
     added = '3.14'
 [function.PyLong_AsUInt64]
     added = '3.14'
+[function.PyLong_AsNativeBytes]
+    added = '3.14'
+[function.PyLong_FromNativeBytes]
+    added = '3.14'
+[function.PyLong_FromUnsignedNativeBytes]
+    added = '3.14'
+[const.Py_ASNATIVEBYTES_DEFAULTS]
```

Comment:
I don't think that you can add the constants to the **ABI** since they are implemented as macros. I suggest to remove these constants.

Reply:
There are a lot of other macros in the file already, though. And the build script hasn't done anything with them

Reply:
They are definitely part of the limited API, which is also defined by this file.

But they are also part of the stable ABI -- we can't change them, even though changing them would *not* be breaking the API.